collections/_release_4_4/entry-highlights-initial-android-editor-support-for-XR-devices.md

```diff
@@ -0,0 +1,21 @@
+---
+type: entry
+section: highlights
+rank: 2
+importance: 1
+anchor: "initial-android-editor-support-for-XR-devices"
+title: "Initial Android editor support for XR devices"
+blockquote: "Make games in VR!"
+text: |
+  Because the Godot Editor itself is a project made with the engine like any other, it can be made available in more unconventional places like the web or on mobile.
+
+  With this release, the first steps have been taken to use OpenXR to transfer the existing Android editor into the context of mixed reality headsets.
+
+  Currently supported on Meta Quest 3 and Quest Pro.
```

Comment:
```suggestion
  Currently supported on Meta Quest 3, Quest Pro and Pico 4.
```

Reply:
@dsnopek Is the pico version of the editor released currently? I can't find it on the pico store or our downloads page

Reply:
Hm. @akien-mga added it to the build scripts in PR https://github.com/godotengine/godot-build-scripts/pull/105 but I guess he hasn't been making builds of it for the beta releases.

It's much further from being ready to go into the Pico store (whereas we already have the editor in the Meta store) but as far as I know, we should be ready to release APKs for users to sideload it themselves.

Reply:
Don't forget the oxford comma.